Shoko.Server/Databases/MySQL.cs

```diff
@@ -633,7 +633,11 @@ public class MySQL : BaseDatabase<MySqlConnection>, IDatabase
             new DatabaseCommand(91, 1, DropMALIndex),
             new DatabaseCommand(92, 1, DropAniDBUniqueIndex),
             new DatabaseCommand(93, 1, "CREATE TABLE `AniDB_Anime_Staff` ( `AniDB_Anime_StaffID` INT NOT NULL AUTO_INCREMENT, `AnimeID` int NOT NULL, `CreatorID` int NOT NULL, `CreatorType` varchar(50) NOT NULL, PRIMARY KEY (`AniDB_Anime_StaffID`) );"),
-            new DatabaseCommand(93, 2, DatabaseFixes.RefreshAniDBInfoFromXML),
+            new DatabaseCommand(93, 2, DatabaseFixes.RefreshAniDBInfoFromXML),
+            new DatabaseCommand(94, 1, "UPDATE VideoLocal_User SET WatchedDate = NULL WHERE WatchedDate = '1970-01-01 00:00:00';"),
+            new DatabaseCommand(94, 2, "ALTER TABLE VideoLocal_User ADD WatchedCount INT NOT NULL DEFAULT 0;"),
+            new DatabaseCommand(94, 3, "ALTER TABLE VideoLocal_User ADD DateTimeUpdated datetime NOT NULL DEFAULT '2000-01-01 00:00:00';"),
```

Comment:
```suggestion
            new DatabaseCommand(94, 3, "ALTER TABLE VideoLocal_User ADD DateTimeUpdated datetime NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP"),
```

Reason for this, it will automatically keep the updated timestamp regardless of user set.